package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@jusbrasil/sixpack-client",
-  "version": "2.3.4",
+  "version": "2.3.5-beta.4",
```

Bump in master branch to `2.3.5`
```suggestion
  "version": "2.3.4",
```